File: programme/compound-interest-calculator/CompoundInterest.java
// Write a programme to calculate compound interest.
import java.util.Scanner;

public class CompoundInterest {
    public static void main(String args[]) {
        try (
                Scanner sc = new Scanner(System.in)) {
            // Input from User
            System.out.print("Enter Principle Amount : ");
            double p = sc.nextInt();
            System.out.print("Enter Time Period : ");
            double t = sc.nextInt();
            System.out.print("Enter Rate of Interest : ");
            double r = sc.nextInt();

            // output
            double amount = (double) (p * Math.pow(1 + (r / 100), t));
            System.out.println();
            System.out.println("Amount : " + Math.round(amount * 100.0) / 100.0);
            System.out.println("Compound Interest : " + Math.round((amount - p) * 100.0) / 100.0);
        }
    }
}

File: programme/compound-interest-calculator/compound-interest.js
// Write a programme to calculate compound interest

function calculateCompoundInterest(amount, time, rate) {
  const finalAmount = amount * (1 + rate / 100) ** time;
  console.log("Amount :", finalAmount.toFixed(2));

  const compoundInterest = finalAmount - amount;
  return compoundInterest.toFixed(2);
}

let amount = 100;
let time = 10;
let rate = 10;

console.log(
  "Compound Interest :",
  calculateCompoundInterest(amount, time, rate)
);
